maint/code-cleanup.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 
-if test ! -z "`which gindent`" ; then
+if hash gindent 2>/dev/null; then
```

Comment:
I don't remember the specifics now, but somewhere it wasn't detecting indent vs. gindent correctly. Making this change made it more reliable. We can drop this if you care. I don't think it matters to me.